src/backend/ProcessLauncher.cpp

```diff
@@ -234,10 +234,10 @@ void ProcessLauncher::onProcessFinished(int exitcode, QProcess::ExitStatus exits
     afterRun();
 }
 
-void ProcessLauncher::beforeRun()
+void ProcessLauncher::beforeRun(const QString& gamePath)
```

Comment:
Could you change this to `game_path`? (Sorry, I just have some perfectionist tendencies, don't take it personally)